docs/config.md

```diff
@@ -55,6 +55,18 @@ $ kubectl az check-apiserver-connectivity
 There is also an option to unset node information from the configuration using
 the `unset-node`/`unset-all`/`unset-current-node` commands.
 
+## Importing configuration
+
+We can also import the node information using `kubeconfig` by talking to the API server:
+
+```bash
+$ kubectl az config import
+$ kubectl az config show
+$ kubectl az use-node aks-agentpool-77471288-vmss000013
+```
+
+The information is stored with node name as key and VMSS instance information as value to avoid talking to the API server on every command.
```

Comment:
```suggestion
The information is stored with node name as key and VMSS instance information as value to avoid talking to be able to continue talking with the node, even if the API server is not working correctly.
```